.github/workflows/workflow.yaml

```
@@ -17,33 +21,32 @@ jobs:
           nix_path: nixpkgs=./nixpkgs/default.nix
       - name: Configure
         run: |
+          [[ ${{ matrix.os }} == macos ]] && EXTRA_CONFIG=--config=macos-nixpkgs
           cat >.bazelrc.local <<EOF
           common --config=ci
-          # The configuration `ci-macos-nixpkgs` is currently undefined.
-          # Replace `ci-macos` by `ci-macos-nixpkgs` when needed.
-          build --config=macos-nixpkgs --config=ci-macos
+          build --config=nixpkgs $EXTRA_CONFIG
           EOF
       - name: Check Bazel version
         run: |
           nix-shell --arg docTools false --pure --run .ci/check-bazel-version
       - name: Build & test
         run: |
-          # Retry if needed due to network flakiness.
-          nix-shell --arg docTools false --pure --run \
-          'cmd="bazel fetch @stackage//..."; $cmd || $cmd || $cmd'
           # XXX 2019-01-22 Disable start script checking on Darwin
           # due to a clash between binutils and clang.
-          # nix-shell --arg docTools false --pure --run \
-          #   './tests/run-start-script.sh --use-nix'
+          [[ ${{ matrix.os }} != macos ]] &&
+            nix-shell --arg docTools false --pure --run \
+              './tests/run-start-script.sh --use-nix'
```

```suggestion
          [[ ${{ matrix.os }} != macos ]] &&
            nix-shell --pure --run \
              './tests/run-start-script.sh --use-nix'
```
Otherwise the case `docTools=true` is untested on CI.

---

In CircleCI this was also `false`, and the `docTools` argument only exists to avoid downloading many Nixpkgs derivations that are not required when not building the documentation. We could remove the `docTools` argument altogether, at the cost of slower CI. But that's orthogonal to moving existing CI jobs to GitHub Actions, so I think we'd want a separate PR.